src/optimizer/join_order_optimizer.cpp

```diff
@@ -663,11 +797,106 @@ unique_ptr<LogicalOperator> JoinOrderOptimizer::RewritePlan(unique_ptr<LogicalOp
 		op = op->children[0].get();
 	}
 	// have to replace at this node
-	//	std::cout << join_tree.second->ToString() << std::endl;
 	parent->children[0] = move(join_tree.second);
 	return plan;
 }
 
+idx_t JoinOrderOptimizer::readable_hash(idx_t table, idx_t col) {
```

Comment:
Can we wrap this into a struct instead of creating this somewhat human-readable integer?

We already have all the code you need for this. The `table, col` pair can be stored in a `ColumnBinding`, and we already have a way to create an `unordered_map` that goes from `ColumnBinding -> whatever`, which is the `column_binding_map_t`, defined in `column_binding_map.hpp`.

Reply:
I see that you're also using it in an `unordered_map`, you can also easily create a `column_binding_set_t` by re using the same code